compiler/backend/cgen.nim

```diff
@@ -1798,7 +1796,7 @@ proc writeModule(m: BModule, pending: bool) =
   var cf = Cfile(nimname: m.module.name.s, cname: cfile,
                   obj: completeCfilePath(m.config, toObjFile(m.config, cfile)), flags: {})
   var code = genModule(m, cf)
-  if code != nil or m.config.symbolFiles != disabledSf:
+  if code != "" or m.config.symbolFiles != disabledSf:
```

Comment:
`code` is a string, so the `$` conversion in the `when hasTinyCBackend` is unnecessary.

Reply:
This is not a big deal, I'm totally OK ignoring this.